src/hexen/r_bmaps.c

```diff
@@ -0,0 +1,710 @@
+//
+// Copyright(C) 1993-1996 Id Software, Inc.
+// Copyright(C) 2005-2014 Simon Howard
+// Copyright(C) 2013-2017 Brad Harding
```

Comment:
Your copyright here and in the header (and on Heretic), please.

Reply:
Done. I've also disabled brightmaps on foggy maps, here's why they shouldn't be applied: [screenshot](https://user-images.githubusercontent.com/21193394/170188315-112fb6da-07e7-459a-abf7-4096a6856f18.png).